crits/settings.py

```diff
@@ -135,6 +136,7 @@
 COL_EMAIL = "email"                                       # main email collection
 COL_EVENTS = "events"                                     # main events collection
 COL_EVENT_TYPES = "event_types"                           # event types for events
+COL_EXPLOIT = "exploit"                                   # exploits
```

Comment:
Yes. There is already a COL_EXPLOITS which is the one used by the mapreduce (at least according to the comment). I still need to go through and remove the old collections that are no longer needed before this is accepted, and add a note to UPDATING that they can be removed from the database.